sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -150,11 +161,31 @@ cdef class StateSampler(object):
           # without the GIL.
           nsecs_ptr = &(<ScopedState>PyList_GET_ITEM(
               self.scoped_states_by_index, self.current_state_index)).nsecs
+          lull_nsecs_ptr = &(<ScopedState>PyList_GET_ITEM(
+              self.scoped_states_by_index, self.current_state_index)).lull_nsecs
```

Comment:
<!--new_thread; commit:95830a027670626ce0f081991d56dac60c8ee8d1; resolved:0-->
Can we use the existing `state_transition_count` property to track lulls, instead of the extra properties you have defined here?

Reply:
I'm sorry, I am not sure how we'd use transition count to track this. Can you give me a snippet for it? Or I could, instead, add a `time_since_last_transition` property to `StateSampler`, and have similar code. Let me know what you think.